qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,261 @@
+        Returns:
+            dict: data object with the circuit's metadata
+                  and the probability for each state in each circuit
+        """
+        if self._simulation_backend:
+            for circuit in circuits:
+                circuit.metadata["is_simulation"] = True
+                circuit.save_probabilities()
+            return execute(circuits, backend=self._simulation_backend, **run_options)
+        else:
+            from qiskit.quantum_info import Statevector
+            import numpy as np
+
+            sim_obj = []
+            for circuit in circuits:
+                circuit.metadata["is_simulation"] = True
+                state_vector = Statevector(circuit)
+                prob_vector = np.multiply(state_vector, state_vector.conjugate())
+                sim_data = {"probabilities": prob_vector, "metadata": circuit.metadata}
+                sim_obj.append(sim_data)
+            return sim_obj
+
+    # pylint: disable = arguments-differ
+    def circuits(self, backend=None):
+        """Return a list of QV circuits, without the measurement instruction
+        Args:
+            backend (Backend): Optional, a backend object.
+        Returns:
+            List[QuantumCircuit]: A list of :class:`QuantumCircuit`s.
+        """
+        circuits = []
+        depth = self._num_qubits
+
+        # continue the trials numbers from previous experiments runs
+        for trial in range(self._previous_trials + 1, self._trials + 1):
+            qv_circ = QuantumVolume(depth, depth, seed=self._rng)
+            qv_circ.metadata = {
+                "experiment_type": self._type,
+                "depth": depth,
+                "trial": trial,
+                "qubits": self.physical_qubits,
+            }
+            circuits.append(qv_circ)
+
+        return circuits
+
+    def transpiled_circuits(self, backend=None, **kwargs):
+        """Return a list of experiment circuits, before and after transpilation.
+
+        Args:
+            backend (Backend): Optional, a backend object to use as the
+                               argument for the :func:`qiskit.transpile`
+                               function.
+            kwargs: kwarg options for the :meth:`circuits` method, and
+                    :func:`qiskit.transpile` function.
+
+        Returns:
+            List[QuantumCircuit]: A list of :class:`QuantumCircuit`s after transpile.
+            List[QuantumCircuit]: A list of :class:`QuantumCircuit`s before transpile.
+
+        Raises:
+            QiskitError: if an initial layout is specified in the
+                         kwarg options for transpilation. The initial
+                         layout must be generated from the experiment.
+
+        .. note:
+            These circuits should be on qubits ``[0, .., N-1]`` for an
+            *N*-qubit experiment. The circuits mapped to physical qubits
+            are obtained via the :meth:`transpiled_circuits` method.
+        """
```

Comment:
What's the difference between this function, and the function that's in the base class?

Reply:
the main difference was that this function returns both circuits before and after transpilation.
after the options PR was merge, this function no longer exists anyway...